include/CLI/Validators.hpp

```diff
@@ -32,37 +32,75 @@ class Option;
 
 ///
 class Validator {
-    friend Option;
-
   protected:
-    /// This is the type name, if empty the type name will not be changed
-    std::string tname;
+    /// This is the validator description
+    std::string description_;
```

Comment:
Could this be removed and all descriptions come from a function? (At one point I did different things based on whether it was a function or not, but I'm not sure you do)